### PR TITLE
cache server error option

### PR DESCRIPTION
**Pull request by omansour**
allow you to set an option disabling caching when server errors

### CODE REVIEW

**Comment by mojoLyon** [comment]
:metal:

File: README.md [hunk 1]
```diff
@@ -188,6 +188,7 @@ m6web_guzzlehttp:
                 service: my_cache_service    # reference to service who implements the cache interface
                 default_ttl: 3600            # defautl ttl for cache entry in seconds
                 use_header_ttl: false        # use the cache-control header to set the ttl
+                cache_server_errors: true    # at false, no server errors will be cached
```
Comment (omansour) [inline README.md]: define an error ! :)

with server_error there is no ambiguity between 5* and 4*. Caching 4* errors can be legit. 

maybe I can add an option cache_client_errors ?

Comment (mikaelrandy) [inline README.md]: ok

**Comment by omansour** [comment]
v1.4.9

**Comment by omansour** [comment]
en fait plutot 1.5.0

**Pull request merged**